integreat_compass/cms/templates/overlay/offer-detail-overlay.html

```diff
@@ -82,6 +82,81 @@ <h3 class="text-xl mt-4 font-semibold">{% translate "More details" %}</h3>
                         </div>
                     </div>
                 </div>
+                <div class="md:flex justify-left gap-12">
```

Comment:
```suggestion
{% if public_version %}
                <div class="md:flex justify-left gap-12">
```
Admittedly  this is only an issue for the root user, which we don't really need to pay attention to, but the root user can access this section also via "Application > Details" (The board members are filtered out via next permission check in the next line) and in this view this section doesn't fully work, because max_comment_length doesn't exist for this view. Again this is not really an issue for non-developer-user, but since there is an easy fix for it by excluding non public versions, I say we add it?

Reply:
Good catch & nice fix!!